docs/userguide/networkingcontainers.md

```diff
@@ -214,7 +214,7 @@ Now, open a shell to your running `db` container:
     --- 172.17.0.2 ping statistics ---
     44 packets transmitted, 0 received, 100% packet loss, time 43185ms
 
-After a bit, use CTRL-C to end the `ping` and you'll find the ping failed. That is because the two container are running on different networks. You can fix that. Then, use CTRL-C to exit the container.
+After a bit, use CTRL-C to end the `ping` and you'll find the ping failed. That is because the two container are running on different networks. You can fix that - use the `exit` command to exit the container.
```

Comment:
Thanks @trdenton  Spotted a typo to fix...also slight tweak to the last sentence.

```markdown
After a bit, use `CTRL-C` to end the `ping` and you'll find the ping failed.
That is because the two containerS are running on different networks. You can
fix that. Then, use the `exit` command to close the container shell.
```